main.py

```diff
@@ -17,49 +18,37 @@ def startup():
     print("startup")
 
 
-client = MongoClient('localhost', 27017)
-
-
 @app.get("/")
 def read_root(request: Request):
     customfcs = request.app.state.custom
     return {"Hello": "customfcs"}
 
 
-class ProductType(str, Enum):
-    FOOD = 'food'
-    EVERYDAY_USE = 'everyday_use'
-
-
-class Product(BaseModel):
-    id: int
-    name: str
-    price: float
-    type: ProductType
-    region: str
-    created_at: date
-
-    @root_validator(pre=True)
-    def validate_id(cls, values):
-        if 'id' not in values:
-            values['id'] = len(products) + 1
-        return values
-
-
-products = []
-
-
-# эдитпоин для добавления нового продукта
-@app.post('/products')
-async def add_product(product: Product):
-    try:
-        products.append(product.dict())
-        return {'status': 'success', 'data': product.dict()}, 201
-    except ValidationError as e:
-        return {'status': 'error', 'message': e.errors()}, 400
+products_data = mongo_db.products.find()
 
 
-# эдитпоинт для получения списка всех продуктов
 @app.get('/products/all')
-async def get_all_products():
-    return {'status': 'success', 'data': products}, 200
+def get_all_products():
+    return products_data
+
+
+# Define the endpoint to add a new Product to MongoDB
+@app.post("/products", status_code=201)
+def add_product_to_db(product: ProductCreate):
+    # Generate a new ObjectId for the product
+    product_id = str(ObjectId())
+
+    # Create a new document with the product data
+    product_data = {
+        'id': int(product_id),
```

Comment:
`id` is created automatically by mongodb during insertion, you don't need to set it by yourself